datasets/asset/dataset.json

```diff
@@ -0,0 +1,267 @@
+{
+    "id": "asset",
```

Comment:
Asset is niet acceptabel als dataset id. Deze term is veel te generiek, uit de datasetid moet blijken wat er in de dataset zit. Zie ook mn eerdere suggestie